stories/index.ts

```diff
@@ -3,6 +3,7 @@ require('./Playground/Playground.story');
 require('./Icons.story');
 
 require('../src/components/ActionsMenuLayout/docs/index.story');
+require('../src/components/ActionsMenuLayout/docs/ActionsMenuLayoutTestStory');
```

Comment:
This should be added on https://github.com/wix/wix-ui-tpa/blob/master/stories/testStories.ts instead of here.

Reply:
oops, updated

Reply:
> Wow very nice!!
> 
> I think that we need to remove the focus onKeyDown event when the key is 'enter' or 'esc'.
> @jonathanadler - What do you think?

imho esc should remove focus and enter should work same as click